x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/use_rule_details_tabs.tsx

```diff
@@ -80,9 +82,17 @@ export const useRuleDetailsTabs = ({
   );
 
   const [pageTabs, setTabs] = useState<Partial<Record<RuleDetailTabs, NavTab>>>(ruleDetailTabs);
-
   const ruleExecutionSettings = useRuleExecutionSettings();
 
+  const { loading: listsConfigLoading, needsConfiguration: needsListsConfiguration } =
+    useListsConfig();
+  const canReadEndpointExceptions = useHasSecurityCapability('crudEndpointExceptions');
```

Comment:
This looks a bit inconsistent, but I am not sure if this should be:
`const canReadEndpointExceptions = useHasSecurityCapability('showEndpointExceptions');`
or
`const canWriteEndpointExceptions = useHasSecurityCapability('crudEndpointExceptions');`

Reply:
Good catch. I think I missed some renaming here. It is indeed checking `crudEndpointExceptions` here.

Reply:
dc4bb74cab9ad1223f67b4111961c44e3efe6afa